test/e2e/containerruntime/container_runtime.go

```diff
@@ -0,0 +1,41 @@
+package containerruntime
+
+import (
+	"os"
+	"strings"
+)
+
+type ContainerRuntime string
+
+func (cr ContainerRuntime) String() string {
+	return string(cr)
+}
+
+const (
+	Docker ContainerRuntime = "docker"
+	Podman ContainerRuntime = "podman"
+)
+
+var runtime ContainerRuntime
+
+func init() {
+	if cr, found := os.LookupEnv("CONTAINER_RUNTIME"); found {
+		switch strings.ToLower(cr) {
+		case Docker.String():
+			runtime = Docker
+		case Podman.String():
+			runtime = Podman
+		default:
+			panic("unknown container runtime")
```

Comment:
nit: how about saying unsupported container runtime and display what the env `CONTAINER_RUNTIME` is set to?